ONSdigital/dp-cantabular-csv-exporter
# add rowCount to produced event

### What

- add rowCount (`row_count`) to produced event
- replaced assistdog dependency with a forked version that supports int32 values (`avro` does not support int values and the original library does not support int32 values)

### How to review

- Make sure code changes make sense
- Make sure unit/component tests are successful and validate the rowCount.

### Who can review

anyone
